samples/suit/ab/README.rst

```diff
@@ -79,77 +329,37 @@ CONFIG_N_BLINKS - Configuration for the number of blinks.
 
 CONFIG_BT_DEVICE_NAME - Configuration for Bluetooth Device Name.
    The sample configuration changes the advertised Bluetooth name.
-   Its default value is ``SUIT SMP Sample``.
+   Its default value is ``SUIT AB Sample``.
 
-Modify partition sizes
-======================
+.. _CONFIG_ID_VAR_BOOT_STATUS:
 
-You can also modify the size and location of the partitions.
-This is done by modifying the values for the desired location and size of the partition in the devicetree :file:`.overlay` files.
+CONFIG_ID_VAR_BOOT_STATUS - Configuration for the Id of Manfiest-controlled variable representing a boot status.
```

Comment:
I think the description formulated this way could create some cofusion"
Maybe: "CONFIG_ID_VAR_BOOT_STATUS - Configuration option allowing to set the ID of the Manifest-controlled variable representing a boot status."

Reply:
Manifest does not use the value from the Kconfig... so I am not sure if it allows to do anything else than sync C logic with manifest logic...

Reply:
Added ` used by the root manifest,` instead.